src/benchmark/tpch_benchmark.cpp

```diff
@@ -50,13 +50,15 @@ int main(int argc, char* argv[]) {
     ("clustering", "Clustering of TPC-H data. The default of --clustering=None means the data is stored as generated "
                    "by the TPC-H data generator. With --clustering=\"Pruning\", the two largest tables 'lineitem' "
                    "and 'orders' are sorted by 'l_shipdate' and 'o_orderdate' for improved chunk pruning. Both are "
-                   "legal TPC-H input data.", cxxopts::value<std::string>()->default_value("None")); // NOLINT
+                   "legal TPC-H input data.", cxxopts::value<std::string>()->default_value("None")) // NOLINT
+    ("use_mmap", "Use storage.json to load TPCH data using the fileformat for chunks that hold their data with mmap."); // NOLINT
```

Comment:
Site note: the user of the benchmarks/Hyrise should not care/be aware about the name of internal files that are used for persistence.